JavaScript/js/messengerDefinition.js

```diff
@@ -1,24 +1,48 @@
-// Jacob's working on this...
+// JD's working on this...
```

Comment:
With that clarification out of the way, I'm _also_ ok with not worrying about assigning code to specific people.
I'm cool with us having the latitude to go and make small changes in code written by other people to enable new functionality _as long as_ the changes are driven by the new functionality, not a desire to make code look the way we prefer it to look.
And if we find we're making large changes and/or deleting whole functions or files, these are things that should be discussed before they happen.